webapp/components/file_upload.jsx

```diff
@@ -198,6 +196,7 @@ class FileUpload extends React.Component {
 
     pasteUpload(e) {
         var inputDiv = ReactDOM.findDOMNode(this.refs.input);
+        var self = this;
```

Comment:
You should just rename the instances of `self` below to `this`. It's not in an event handler so there's no change that `this` will be rebound and using `self` will also cause ESLint to print warnings